### PR TITLE
Update to netstandard2.1

### PR DESCRIPTION
**Pull request by tabman83**
This PR updates the eshopworld.devops package to netstandard2.1

### CODE REVIEW

**Comment by VaclavK** [comment]
We should upgrade all extensions packages imho

File: src/Eshopworld.DevOps/Eshopworld.DevOps.csproj [hunk 1]
```diff
@@ -1,12 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>4.2.0</Version>
-    <PackageReleaseNotes>
-      Changed the order of configuration providers. Now the order is: configuration files, Key Vault secrets, environment variables (the later overwrites the former).
-    </PackageReleaseNotes>
+    <Version>5.0.0-preview1</Version>
```
Comment (dguerin) [inline src/Eshopworld.DevOps/Eshopworld.DevOps.csproj]: What's the need for a preview tag here?

**Comment by dguerin** [comment]
> @tabman83 looks like the build has failed for you. Can you take a look?

the nuget restore issue with the build is fixed but there is another. Didn't have time to look at it today but I can in the morning

File: src/Eshopworld.DevOps/Eshopworld.DevOps.csproj (hunk 1 above)
Comment (tabman83) [reply src/Eshopworld.DevOps/Eshopworld.DevOps.csproj]: I was told all the packages that incur a .net core bump should have a preview tag until we are confident enough about the upgrade.

**Pull request merged**